.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+jdk:
+  - oraclejdk8
+  - oraclejdk7
+
```

Comment:
Do you think we should add the openjdks as well? I doubt we'd see much of a difference, but there are some differences and so we might want to include them for the random edge case.